annotations/static/annotations/js/annotators/text.js

```diff
@@ -100,8 +101,9 @@ var ConceptSearch = {
                 self.error = true;
                 self.searching = false;
             });
-        }
-    },
+        },
+        },
```

Comment:
You may want to indent this 4 spaces to the left.

Reply:
yes